src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -34,16 +43,34 @@ type BorrowInputProps = {
   collateralAssetId: AssetId
   depositAmount: string | null
   onDepositAmountChange: (value: string) => void
+  collateralAccountId: AccountId
+  borrowAccountId: AccountId
+  onCollateralAccountIdChange: (accountId: AccountId) => void
+  onBorrowAccountIdChange: (accountId: AccountId) => void
+  borrowAsset: Asset | null
+  setBorrowAsset: (asset: Asset) => void
 }
 
 export const BorrowInput = ({
   collateralAssetId,
   depositAmount,
   onDepositAmountChange,
+  collateralAccountId,
+  borrowAccountId,
+  onCollateralAccountIdChange: handleCollateralAccountIdChange,
+  onBorrowAccountIdChange: handleBorrowAccountIdChange,
+  borrowAsset,
+  setBorrowAsset,
 }: BorrowInputProps) => {
   const translate = useTranslate()
   const history = useHistory()
 
+  const { data: lendingSupportedAssets = [] } = useLendingSupportedAssets()
```

Comment:
the coaleasce here creates a new array every frame where data is undefined, we should memoise the coalescing

Reply:
Might as well remove the default altogether, more safu that way 487730f4d0